### DISCUSSION TITLE
How to open .pb file

### DISCUSSION THREAD

**Discussion by trevitran** [deepmodeling/deepmd-kit#1271] [Q&A]
I am currently freezing the dataset from examples/water/se_e2_a and was wondering how I can open the graph.pb file to see the frozen model/graph.  I'm using DeePMD Kit version  2.0.2, Tensorflow 2.6.0, Python 2.7.5. Thank you.

**Comment by wanghan-iapcm** [comment]
@denghuilu  could you provide suggestions?

**Reply by denghuilu** [reply]
One can use the tensorboard tool to visualize the frozen pb model.
Check this https://blog.csdn.net/u014061630/article/details/80557028 or https://github.com/akshaybhor733/.pb-file-Tensorboard/blob/master/tensorboard.pb.py.

See if you can find what you want.
